### PR TITLE
Added ico amd bmp extensions

### PR DESCRIPTION
**Pull request by aurangzaib048**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,6 +1,6 @@
-all: wasm-thumbnail-py/src/wasm_thumbnail/data/wasm_thumbnail.wasm
+all: wasm-thumbnail-py/wasm_thumbnail
 
 wasm-thumbnail-py/src/wasm_thumbnail/data/wasm_thumbnail.wasm: wasm-thumbnail/src/lib.rs
 	cd wasm-thumbnail && cargo build --release
-	cp wasm-thumbnail/target/wasm32-unknown-unknown/release/wasm_thumbnail.wasm wasm-thumbnail-py/src/wasm_thumbnail/data/
+	cp wasm-thumbnail/target/wasm32-unknown-unknown/release/wasm_thumbnail.wasm wasm-thumbnail-py/wasm_thumbnail/data
```
Comment (fallaciousreasoning) [inline Makefile:5]: I think something is slightly off with this, as running `make` will fail now

Comment (aurangzaib048) [reply Makefile:5]: If you create the dir like `wasm-thumbnail-py/wasm_thumbnail/data/` that will be fixed

Comment (fallaciousreasoning) [reply Makefile:5]: Perhaps the script should automatically generate that directory?

Comment (aurangzaib048) [reply Makefile:5]: Not yet, But I have to add some cmd to make it possible.

**Comment by fallaciousreasoning** [comment]
@aurangzaib048 when do you think we'll be able to test this in staging?

**Pull request merged**